data_generators/generators.py

```diff
@@ -1,5 +1,5 @@
 import os
-from random import sample, randint
```

Comment:
Was randint deleted on accident? It is still being used/needed throughout this file.